src/main/java/com/test/sb/model/Review.java

```diff
@@ -0,0 +1,24 @@
+package com.test.sb.model;
```

Comment:
```suggestion
package com.test.sb.dto;
```
in fact - this is a Dto that we will later use to form entities and save them to db